src/Laravel/Casts/RutCast.php

```
@@ -0,0 +1,65 @@
+    }
+
+    public function get(Model $model, string $key, mixed $value, array $attributes): ?Rut
+    {
+        if ($value === null || $value === '') {
+            return null;
+        }
+
+        return Rut::parse((string) $value)->quiet();
+    }
+
+    public function set(Model $model, string $key, mixed $value, array $attributes): ?string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)